src/Passable.js

```diff
@@ -40,21 +40,15 @@ class Passable {
             return;
         }
 
-        const lastIndex: number = args.length - 1;
-        let callback: Function;
+        this.res.initFieldCounters(fieldName);
 
-        if (typeof args[lastIndex] === 'function') {
-            callback = args[lastIndex];
-        } else {
+        if (typeof test !== 'function') {
```

Comment:
Why this not the first check int this function? Why you want to `initFieldCounters` if there is nothing to test?